products/cakephp.md

```diff
@@ -22,9 +22,16 @@ auto:
 # For a given major version, the last three minor versions have security support.
 # Do not forget to document the codename in the product's description when adding a new major version.
 releases:
--   releaseCycle: "4.4"
+-   releaseCycle: "5.0"
     eol: false
     support: true
+    releaseDate: 2023-09-09
+    latest: "5.0.0"
+    latestReleaseDate: 2023-09-09
+
+-   releaseCycle: "4.4"
+    eol: 2026-09-09
```

Comment:
It's strange to have the `eol` date set for 4.4, but not for 4.3 or 4.2. As per https://github.com/cakephp/cakephp/wiki#important-future-release-dates:

> As [previously communicated](https://bakery.cakephp.org/2023/01/01/cakephp_3_end_of_life_3105_released.html), 4.x will:
>
>  - Receive bug fixes and feature releases until September 2025.
>  - Receive security fixes until September 2026.

So let's use 2026-09-09 also for 4.3 and 4.2.